vispy/glsl/misc/spatial-filters.frag

```diff
@@ -34,29 +44,60 @@ filter2D_radius1(sampler2D texture, sampler2D kernel, float index, vec2 uv, vec2
     return filter1D_radius1(kernel, index, f.y, t0, t1);
 }
 
+float lerp(sampler2D kernel, vec2 uv) {
```

Comment:
just a point of curiosity, why `lerp`? Linear interp? Maybe we should make more explicit like `unpack_interpolate(...)`